### DISCUSSION TITLE
How to delete repository with GraphQL

### DISCUSSION THREAD

**Discussion by heyallan** [community/community#24368] [Apps, API and Webhooks]
<p>I have managed to create, edit, and delete a repository with REST API.<br>
Now I’m trying to do the same with GraphQL, but I can’t find info on how to delete.<br>
Where is the delete documentation?</p>
<p>Reference: <a href="https://docs.github.com/en/free-pro-team@latest/graphql/reference/mutations" rel="noopener nofollow ugc">https://docs.github.com/en/free-pro-team@latest/graphql/reference/mutations</a></p>

**Comment by byrneh** [comment, marked as answer]
<p><a class="mention" href="/heyallan">@heyallan</a>, you are looking on the correct place, but this delete repository mutation does not exist in GraphQL API at present</p>

**Comment by tik9** [comment]
Are plans there to make a mutation that deletes repos?
